content/docs/command-reference/status.md

```diff
@@ -157,44 +157,45 @@ that.
 
 ```dvc
 $ dvc status
-bar.dvc:
-        changed deps:
-                modified:      bar
-        changed outs:
-                not in cache:      foo
-foo.dvc:
-        changed outs:
-                deleted:      foo
-        changed checksum
-prepare.dvc:
-        changed outs:
-                new:      bar
-        always changed
+foo:
+	changed deps:
+		modified:           bar
+	changed outs:
+		not in cache:       foo
+bar:
+	changed outs:
+		deleted:            bar
+	changed checksum
+baz.dvc:
+	changed outs:
+		new:                baz
+	always changed
 ```
 
-This shows that for stage `bar.dvc`, the dependency `foo` and the
-<abbr>output</abbr> `bar` have changed. Likewise for `foo.dvc`, the dependency
-`foo` has changed, but no output has changed.
+This shows that for stage `foo`, the dependency `bar` and the
+<abbr>output</abbr> `foo` have changed, and for stage `bar`, the output `bar`
+has changed, but no dependency has changed. Likewise for `baz.dvc`, the output
+`baz` tracked by it has changed.
 
 ## Example: Specific files or directories
 
 `dvc status` only checks the tracked data corresponding to any given `targets`:
 
 ```dvc
-$ dvc status foo.dvc dobar
-foo.dvc:
-  changed outs:
-          deleted:      foo
-  changed checksum
+$ dvc status dofoo.dvc dobar
+dofoo.dvc:
+	changed outs:
+		deleted:            dofoo
+	changed checksum
 dobar:
```

Comment:
> Continuation of https://github.com/iterative/dvc.org/pull/1952#pullrequestreview-536131974

No, I meant use "do{x}" for stage names (and referred to the previous code block in line 160+). Regular foo, bar, baz, etc. is OK for .dvc files.

Reply:
I'll fix it.
```suggestion
$ dvc status dofoo.dvc dobar
foo.dvc:
	changed outs:
		deleted:            foo
	changed checksum
dobar:
```